visualization/mayavi_vis_utils.py

```diff
@@ -0,0 +1,116 @@
+"""
+Utilities for rendering camera frustums and 3d point clouds using Mayavi mlab.
+
+Author: John Lambert
+"""
+
+import argparse
+from typing import List
+
+import mayavi
+import numpy as np
+from colour import Color
+from gtsam import Cal3Bundler, Pose3
+from mayavi import mlab
+
+
+from gtsfm.common.view_frustum import ViewFrustum
+
+
+def draw_point_cloud_mayavi(
+    args: argparse.Namespace, fig: mayavi.core.scene.Scene, point_cloud: np.ndarray, rgb: np.ndarray
```

Comment:
Is only sphere_radius required? No need pass all arguments.

Reply:
same for other usages.

Reply:
i changed it in just two functions. let me know if you think i should change it in more places also